packages/node-xmpp-server/test/integration/C2S/WebSocket.js

```diff
@@ -21,7 +21,7 @@ server.on('connection', function (connection) {
   })
 })
 
-describe.skip('C2S WebSocket server client', function () {
```

Comment:
this was temporary disabled during the move to WebSocket final RFC and we forgot to re-enable it